spark/templates/configmap.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if .Values.Livy.Enabled }}
```

Comment:
There are things to change inside this config map that refer to the old values names

Reply:
added a `checksum/configmap`